src/crawlee/events/_local_event_manager.py

```diff
@@ -24,7 +24,12 @@
 
 @docs_group('Classes')
 class LocalEventManager(EventManager):
-    """Local event manager for emitting system info events."""
+    """Local event manager for emitting system info events.
```

Comment:
The fact that it emits `SystemInfo` probably doesn't need to be on the first line, does it?